spec/factories/organization_campaigns.rb

```diff
@@ -16,8 +16,17 @@
   factory(:organization_campaign) do
     organization
     campaign
-    description 'MyString'
+    description 'Share Your Christmas 2015 for United Way'
     reminder_date Date.today + 7
     donation_deadline Date.today + 14
   end
+
+  factory(:organization_campaign_no_recipients, class: OrganizationCampaign) do
```

Comment:
Could be a trait again, but this would be a little more complex. You'd assign it to a campaign that has a trait of no recipients.

By default, I think the campaigns don't have recipients so as-is I don't think this is doing anything different than the code above (other than using a different campaign with it)